src/setup.py

```diff
@@ -8,14 +8,35 @@
 
 import pyfingerprint
 
+with open('README.md', 'r') as fh:
```

Comment:
No abbreviations for variable names please

Reply:
Sure! Changed that